traits/ctraits.c

```diff
@@ -5107,20 +5115,23 @@ PyDoc_STRVAR(
     "    is modified.\n");
 
 PyDoc_STRVAR(
-    property_doc,
-    "property()\n"
-    "property(get, get_n, set, set_n, validate, validate_n)\n"
+    _get_property_doc,
+    "_get_property()\n"
     "\n"
-    "Get or set property fields for this trait.\n"
+    "Get the property fields for this trait.\n"
     "\n"
     "When called with no arguments on a property trait, this method returns a\n"
```

Comment:
The "called with no arguments" part of the docstring is redundant at this point.